src/lazy/placeholders/hydrate.tsx

```diff
@@ -13,8 +14,10 @@ export const PlaceholderFallbackHydrate = ({
     <>
       <input type="hidden" data-lazy-begin={id} />
       {content.map((el: HTMLElement, i: number) => {
-        const { tagName = '', childNodes = [] } = el;
-        const props = { key: String(i) };
+        const { tagName = '', childNodes = [], attributes = [] } = el;
+        const props = Array.from(attributes).reduce(attrToProp, {
+          key: String(i),
+        });
```

Comment:
Discovered that by not replicating the attributes on first level tags, react logs a warning. So we should try to put most attrs back